resource/META-INF/plugin.xml

```diff
@@ -1,6 +1,8 @@
 <idea-plugin>
   <id>com.zenuml.jetbrains</id>
   <name>ZenUML support</name>
+  <version>0.0.4</version>
+  <idea-version since-build="14.1"/>
```

Comment:
can not find this 14.1 build number,  which version are we targeting at?

Reply:
@MrCoder  
Check this out. https://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html

My assumption is the build number should be in the page above.

Reply:
Looks you are right. Will update it to `162`.